### PR TITLE
Refactor `ContractAddress` method in config structs

### PR DESCRIPTION
**Pull request by lukasz-zimnoch**
`ContractAddress` method in both Ethereum's and Celo's config structs should return a plain `common.Address` type instead of a pointer. This should eliminate the need for nil-checks in the client code.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,6 @@ replace github.com/urfave/cli => github.com/keep-network/cli v1.20.0
 require (
 	github.com/allegro/bigcache v1.2.1 // indirect
 	github.com/aristanetworks/goarista v0.0.0-20190924011532-60b7b74727fd // indirect
-	github.com/btcsuite/btcd v0.20.1-beta // indirect
```
Comment (pdyraga) [inline go.mod:10]: What is happening here? Seems like an unrelated change.

Comment (lukasz-zimnoch) [reply go.mod:10]: `go mod tidy` cleaned it up.

**Review by pdyraga — APPROVED** [review]
